tools/analysis/maian/signing_transactions.md

```diff
@@ -0,0 +1,26 @@
+### Overview
+
+The point of this document is to analyze how different web3 providers handle the eth_sendTransaction method.
+
+Method [eth_sendTransaction](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sendtransaction) creates a new message call transaction or a contract creation, if the data field contains code, and signs it using the account specified in "from" field.
+
+The main focus of the analysis is detecting the source of the private key used to sign the transaction across multiple JSON RPC API implementations.
+
+### Comparison
+
+1. [GETH](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-personal) (historically). This is the implementation required by the MAIAN tool.
+    - Private key is provided by [eth_personal_importRawKey](https://web3js.readthedocs.io/en/v1.2.7/web3-eth-personal.html#importrawkey). Password is submitted together with the private key, and will be required to unlock this key in the next steps.
+    - Account has to be [unlocked](https://web3js.readthedocs.io/en/v1.2.7/web3-eth-personal.html#unlockaccount) in order to be used to signing the transaction. In order to unlock the account user has to submit the password it is associated with.
+    - eth_sendTransaction can be used. If the account provided in the "from" field is unlocked its private key will be used to create a signature.
```

Comment:
I would put this last, also maybe mentioning that this would be the least used case